packages/service-library/src/servicelib/redis.py

```diff
@@ -14,13 +14,13 @@
 from redis.asyncio.lock import Lock
 from redis.asyncio.retry import Retry
 from redis.backoff import ExponentialBackoff
-from servicelib.retry_policies import RedisRetryPolicyUponInitialization
-from servicelib.utils import logged_gather
 from settings_library.redis import RedisDatabase, RedisSettings
 from tenacity import retry
 
 from .background_task import periodic_task, start_periodic_task, stop_periodic_task
 from .logging_utils import log_catch, log_context
+from .retry_policies import RedisRetryPolicyUponInitialization
```

Comment:
do you have a tool to find the absolute vs relative or that was by chance?

Reply:
nope. I created a script to do all the replacements here I can imagine how to do something simialr for the import. will give it a try